code/modules/clothing/suits/armor.dm

```diff
@@ -525,6 +525,12 @@
 	body_parts_covered = UPPER_TORSO | LOWER_TORSO | LEGS | ARMS
 	armor = list(melee = 50, bullet = 30, laser = 30, energy = 15, bomb = 40, bio = 0, rad = 0)
 
+/obj/item/clothing/suit/armor/vest/idris
+	name = "Idris reclamation unit coat "

```

Comment:
Make the capitilization of `reclamation unit` consistent. It's either a proper name or a noun.